source/reporters/ut_documentation_reporter.tpb

```diff
@@ -151,7 +153,8 @@ create or replace type body ut_documentation_reporter is
 
       if a_item.warnings is not null and a_item.warnings.count > 0 then
         for i in 1 .. a_item.warnings.count loop
-          self.print_text('  ' || i || ') ' || a_item.path);
+          l_warning_index := l_warning_index + 1;
+          self.print_text('  ' || l_warning_index || ') ' || a_item.path);
```

Comment:
Can you make both `error_no` and `warning_no` uniformly indented
```
lpad(l_warning_index,5)
```
I don't expect any report to have more than 9999 errors or warnings in it.